src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -89,7 +109,9 @@
                 runningStatus: "",
                 runningKey: "",
                 disableRun: false,
-                parameterValues: []
+                parameterValues: [],
+                changeLogMessageValue: "",
+                changeLogTypeValue: ""
```

Comment:
Like other values, I think these fields should be reset when the selected report changes